.travis.yml

```diff
@@ -1,36 +1,36 @@
 sudo: required
 language: java
 jdk:
-  - oraclejdk8
-  - oraclejdk11
+  - openjdk8
+  - openjdk11
 addons:
   postgresql: "9.4"
   firefox: "60.3.0esr"
 cache:
   directories:
     - $HOME/.m2
+services:
+  - xvfb
 
 install:
   - pushd `pwd`
   - cd $HOME/build
   - if [[ -z $GFW_BRANCH ]]; then GFW_BRANCH=master; fi
   - git clone --depth=1 --branch=$GFW_BRANCH https://github.com/terasolunaorg/terasoluna-gfw.git terasolunaorg/terasoluna-gfw
   - cd terasolunaorg/terasoluna-gfw
-  - sh ./mvn-build-all.sh install -P default -Dmaven.test.skip=true -Dmaven.javadoc.skip=true -Dsource.skip=true
+  - sh ./mvn-build-all.sh -B install -P default -Dmaven.test.skip=true -Dmaven.javadoc.skip=true -Dsource.skip=true
```

Comment:
**not required.**

It seems better to change to `sh ./mvn-build-all.sh install -B -P default ...`.

```
mvn [command] [option] <- good
mvn [option(-B)] [command] [option(-P)] <- hmm...
```